rest/server_context.go

```diff
@@ -582,6 +582,8 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 		}
 	}
 
+	base.Infof(base.KeyAll, "db=%s Delta Sync: %t", dbName, deltaSyncOptions.Enabled)
```

Comment:
Probably want to either align this syntax with the other startup messages (e.g. 'Delta sync enabled=%d for database %s'), or match the format being used for contextual db logging.

Reply:
I'll switch it to `Delta sync enabled=%t for database %s` for now.